extension/.vscode/settings.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Is the extension project intended to be open with vscode?

Reply:
I do some of my C# work in VS Code now instead of VS2019. This ensures that I have the extension setup for plugins.